### ISSUE TITLE
[eFSR] Accessibility testing color and contrast bug

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#59517]
## Issue Description

The new Delete button fails the [contrast ratio on hover](https://www.whocanuse.com/?bg=f9dede&fg=cd2026&fs=16&fw=).

![Screenshot 2023-05-26 at 3.26.10 PM.png](https://images.zenhubusercontent.com/63ea96045c6deec1c9a25425/263691e1-c1c1-4f54-ab2c-a7a7bf4ad65f)
![Screenshot 2023-05-26 at 3.35.22 PM.png](https://images.zenhubusercontent.com/63ea96045c6deec1c9a25425/7cab274c-5aaf-4ddf-83c4-9a6104a17909)

![Screenshot 2023-05-26 at 3.26.47 PM.png](https://images.zenhubusercontent.com/63ea96045c6deec1c9a25425/cd20fe9d-4e14-4b9f-992b-3c72ede02b28)

---
## Tasks
- [x] Fix the failing color contrast ratio for the new delete button's hover state.

## Acceptance Criteria
- [x] Delete button has sufficient contrast ratio on hover. 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

_Originally posted by @josephrlee in https://github.com/department-of-veterans-affairs/va.gov-team/issues/57589#issuecomment-1565026381_
